test-workloads/src/bin/tellus.rs

```diff
@@ -371,6 +394,56 @@ extern "C" fn kernel_init(hart_id: u64, fdt_addr: u64) {
     );
     reclaim_pages(state_pages_base, tvm_create_pages);
 
+    let msg = SbiMessage::Pmu(PmuFunction::GetNumCounters);
+    let num_counters = unsafe { ecall_send(&msg).expect("Tellus - GetNumCounters returned error") };
```

Comment:
missing safety comment, but I'd suggest wrapping this in a safe function at it seems like it'll always be safe.